src/main/java/org/reactome/release/qa/check/CoV2InfectionPathwayEventCheck.java

```diff
@@ -0,0 +1,188 @@
+            }
+
+            // inferredFrom being null is a proxy for curator being 'done' with instance. Therefore, if it is null and the
+            // issue is being reported, it indicates they likely made an error.
+            // If literatureReference attribute does not contain at least 1 litRef from 2020.
+            if (cov2Event.getAttributeValue(ReactomeJavaConstants.inferredFrom) == null && !hasRecentLiteratureReference(cov2Event)) {
+                issues.add("Does not contain a 2020 literature reference");
+            }
+            // Summation text was updated near end of release cycle where CoV-1-to-CoV-2 projections happened.
+            // If it didn't have the updated text, there are two reasons for it, outlined below.
+            if (!hasCorrectSummationFormat(cov2Event)) {
+                if (cov2Event.getAttributeValue(ReactomeJavaConstants.inferredFrom) == null) {
+                    issues.add("InferredFrom is empty but instance has COVID inference summation text");
+                } else {
+                    issues.add("InferredFrom is populated but instance does NOT have COVID inference summation text.");
+                }
+            }
+            if (issues.size() > 0) {
+                report.addLine(getReportLine(cov2Event, String.join("|",issues)));
+            }
+        }
+        report.setColumnHeaders(getColumnHeaders());
+        return report;
+    }
+
+    /**
+     * Checks if instance created dateTime is before most recent modified instance's dateTime. Also checks if
+     * any modified instance is present -- it should be given that there was at least 1 required modification from the
+     * CoV-1-to-CoV-2 merge.
+     * @param cov2Event - GKInstance, Event contained within 'SARS-CoV-2 Infection' Pathway.
+     * @return - boolean, true if modification property looks correct, false if not.
+     * @throws Exception, thrown by MySQLAdaptor.
+     */
+    private boolean hasRecentlyModifiedSummation(GKInstance cov2Event) throws Exception {
+       for (GKInstance summation : (Collection<GKInstance>) cov2Event.getAttributeValuesList(ReactomeJavaConstants.summation)) {
+            GKInstance createdInst = (GKInstance) summation.getAttributeValue(ReactomeJavaConstants.created);
+            List<GKInstance> modifieds = summation.getAttributeValuesList(ReactomeJavaConstants.modified);
+            if (modifieds.size() > 0) {
+                // At time of writing (August 2020), nothing is reported from this clause, but it is a good check.
+                GKInstance mostRecentModifiedInst = modifieds.get(modifieds.size() - 1);
+                // Only check Year/Month/Day since the changes took place more than a day before Curators got to look at them in gk_central.
+                LocalDate createdDateTime = getDateTimeFromInstance(createdInst);
+                LocalDate modifiedDateTime = getDateTimeFromInstance(mostRecentModifiedInst);
```

Comment:
Should the modified instance checked be the most recent or the least recent?  I'm just thinking that it's the least recent that will be older and could have a date before the created time while the most recent may not.

Reply:
This test was actually only created in response to the single 'cov-inference' slice database merge - apparently something wonky happened with curator summation instances and the inference ones (summations being copied over during inference was applied for this project). I don't know for sure, but I think it was related to curations that weren't committed to gk_central before the merge event.

Anyways - this was the requested format of the test, to compare the most recent modified value